src/components/cl/hier/cl_hier_team.c

```diff
@@ -74,18 +79,51 @@ UCC_CLASS_INIT_FUNC(ucc_cl_hier_team_t, ucc_base_context_t *cl_context,
             hs->n_tls = 0;
             tls       = &lib->cfg.sbgp_tls[i].array;
```

Comment:
qq: is orders of TLS in this array always will be same? do you see any issues if there is a mismatched list of TLS selected for team on different ranks?

Reply:
it matters, there is separate fix for this https://github.com/openucx/ucc/pull/717, but it was rejected since the bigger change in UCC is needed to detect mismatches for every CL and core